### PR TITLE
HADOOP-17163. ABFS: Adding debug log for rename failures

### PR DESCRIPTION
**Pull request by bilaharith**
Adding debug log for ABFS rename failures.

### CODE REVIEW

File: hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystem.java [hunk 1]
```diff
@@ -323,6 +323,7 @@ public boolean rename(final Path src, final Path dst) throws IOException {
       abfsStore.rename(qualifiedSrcPath, qualifiedDstPath);
       return true;
     } catch(AzureBlobFileSystemException ex) {
+      LOG.debug("Rename operation failed", ex);
```
Comment (snvijaya) [inline hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystem.java:326]: Space needed after "failed"

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |  35m 36s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -1 :x: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ trunk Compile Tests _ |
| +1 :green_heart: |  mvninstall  |  20m 29s |  trunk passed  |
| +1 :green_heart: |  compile  |   0m 35s |  trunk passed with JDK Ubuntu-11.0.8+10-post-Ubuntu-0ubuntu118.04.1  |
| +1 :green_heart: |  compile  |   0m 31s |  trunk passed with JDK Private Build-1.8.0_252-8u252-b09-1~18.04-b09  |
| +1 :green_heart: |  checkstyle  |   0m 23s |  trunk passed  |
| +1 :green_heart: |  mvnsite  |   0m 36s |  trunk passed  |
| +1 :green_heart: |  shadedclient  |  15m  5s |  branch has no errors when building and testing our client artifacts.  |
| -1 :x: |  javadoc  |   0m 27s |  hadoop-azure in trunk failed with JDK Ubuntu-11.0.8+10-post-Ubuntu-0ubuntu118.04.1.  |
| +1 :green_heart: |  javadoc  |   0m 26s |  trunk passed with JDK Private Build-1.8.0_252-8u252-b09-1~18.04-b09  |
| +0 :ok: |  spotbugs  |   0m 57s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| +1 :green_heart: |  findbugs  |   0m 55s |  trunk passed  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 31s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 30s |  the patch passed with JDK Ubuntu-11.0.8+10-post-Ubuntu-0ubuntu118.04.1  |
| +1 :green_heart: |  javac  |   0m 30s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 25s |  the patch passed with JDK Private Build-1.8.0_252-8u252-b09-1~18.04-b09  |
| +1 :green_heart: |  javac  |   0m 25s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   0m 16s |  the patch passed  |
| +1 :green_heart: |  mvnsite  |   0m 28s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  shadedclient  |  14m 35s |  patch has no errors when building and testing our client artifacts.  |
| -1 :x: |  javadoc  |   0m 24s |  hadoop-azure in the patch failed with JDK Ubuntu-11.0.8+10-post-Ubuntu-0ubuntu118.04.1.  |
| +1 :green_heart: |  javadoc  |   0m 24s |  the patch passed with JDK Private Build-1.8.0_252-8u252-b09-1~18.04-b09  |
| +1 :green_heart: |  findbugs  |   0m 59s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   1m 25s |  hadoop-azure in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 31s |  The patch does not generate ASF License warnings.  |
|  |   |  97m 44s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.40 ServerAPI=1.40 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2180/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/2180 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient findbugs checkstyle |
| uname | Linux 344c2baa326d 4.15.0-60-generic #67-Ubuntu SMP Thu Aug 22 16:55:30 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 05b3337a460 |
| Default Java | Private Build-1.8.0_252-8u252-b09-1~18.04-b09 |
| Multi-JDK versions | /usr/lib/jvm/java-11-openjdk-amd64:Ubuntu-11.0.8+10-post-Ubuntu-0ubuntu118.04.1 /usr/lib/jvm/java-8-openjdk-amd64:Private Build-1.8.0_252-8u252-b09-1~18.04-b09 |
| javadoc | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2180/1/artifact/out/branch-javadoc-hadoop-tools_hadoop-azure-jdkUbuntu-11.0.8+10-post-Ubuntu-0ubuntu118.04.1.txt |
| javadoc | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2180/1/artifact/out/patch-javadoc-hadoop-tools_hadoop-azure-jdkUbuntu-11.0.8+10-post-Ubuntu-0ubuntu118.04.1.txt |
|  Test Results | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2180/1/testReport/ |
| Max. process+thread count | 414 (vs. ulimit of 5500) |
| modules | C: hadoop-tools/hadoop-azure U: hadoop-tools/hadoop-azure |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2180/1/console |
| versions | git=2.17.1 maven=3.6.0 findbugs=4.0.6 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 32s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| -1 :x: |  test4tests  |   0m  0s |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
||| _ trunk Compile Tests _ |
| +1 :green_heart: |  mvninstall  |  19m 24s |  trunk passed  |
| +1 :green_heart: |  compile  |   0m 36s |  trunk passed with JDK Ubuntu-11.0.8+10-post-Ubuntu-0ubuntu118.04.1  |
| +1 :green_heart: |  compile  |   0m 33s |  trunk passed with JDK Private Build-1.8.0_252-8u252-b09-1~18.04-b09  |
| +1 :green_heart: |  checkstyle  |   0m 24s |  trunk passed  |
| +1 :green_heart: |  mvnsite  |   0m 35s |  trunk passed  |
| +1 :green_heart: |  shadedclient  |  14m 16s |  branch has no errors when building and testing our client artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 29s |  trunk passed with JDK Ubuntu-11.0.8+10-post-Ubuntu-0ubuntu118.04.1  |
| +1 :green_heart: |  javadoc  |   0m 26s |  trunk passed with JDK Private Build-1.8.0_252-8u252-b09-1~18.04-b09  |
| +0 :ok: |  spotbugs  |   0m 54s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| +1 :green_heart: |  findbugs  |   0m 52s |  trunk passed  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 29s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 29s |  the patch passed with JDK Ubuntu-11.0.8+10-post-Ubuntu-0ubuntu118.04.1  |
| +1 :green_heart: |  javac  |   0m 29s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 24s |  the patch passed with JDK Private Build-1.8.0_252-8u252-b09-1~18.04-b09  |
| +1 :green_heart: |  javac  |   0m 24s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   0m 16s |  the patch passed  |
| +1 :green_heart: |  mvnsite  |   0m 28s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  shadedclient  |  14m 25s |  patch has no errors when building and testing our client artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 25s |  the patch passed with JDK Ubuntu-11.0.8+10-post-Ubuntu-0ubuntu118.04.1  |
| +1 :green_heart: |  javadoc  |   0m 23s |  the patch passed with JDK Private Build-1.8.0_252-8u252-b09-1~18.04-b09  |
| +1 :green_heart: |  findbugs  |   0m 57s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   1m 24s |  hadoop-azure in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 29s |  The patch does not generate ASF License warnings.  |
|  |   |  60m 23s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.40 ServerAPI=1.40 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2180/4/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/2180 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient findbugs checkstyle |
| uname | Linux 717737f0c532 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 0277856738a |
| Default Java | Private Build-1.8.0_252-8u252-b09-1~18.04-b09 |
| Multi-JDK versions | /usr/lib/jvm/java-11-openjdk-amd64:Ubuntu-11.0.8+10-post-Ubuntu-0ubuntu118.04.1 /usr/lib/jvm/java-8-openjdk-amd64:Private Build-1.8.0_252-8u252-b09-1~18.04-b09 |
|  Test Results | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2180/4/testReport/ |
| Max. process+thread count | 420 (vs. ulimit of 5500) |
| modules | C: hadoop-tools/hadoop-azure U: hadoop-tools/hadoop-azure |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2180/4/console |
| versions | git=2.17.1 maven=3.6.0 findbugs=4.0.6 |
| Powered by | Apache Yetus 0.13.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

File: hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystem.java (hunk 1 above)
Comment (bilaharith) [reply hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystem.java]: Done

**Comment by mukund-thakur** [comment]
I think this should be warning rather that debug.

**Pull request merged**